.github/workflows/release-mac.yaml

```diff
@@ -15,34 +15,37 @@ on:
         description: "Version to tag release artifacts."
         required: true
         type: string
+  pull_request:
+    branches:
+      - master
 
 jobs:
   build:
     runs-on: macos-11
     if: "!contains(github.event.head_commit.message, 'release skip')"
     steps:
-      - name: Configure variables
-        id: vars
-        run: |
-          # Configure repo based on pushed tag if set
-          if [ -z "${GITHUB_REF}" ]; then
-            ref="${GITHUB_REF}"
-            tag=${GITHUB_REF/refs\/tags\//}
-          else
-            # Otherwise the workflow must have been manually triggered with inputs
-            ref="${{ inputs.release_branch }}"
-            tag="${{ inputs.version_tag }}"
-          fi
-          echo "ref=$ref" >> $GITHUB_OUTPUT
-          echo "tag=$tag" >> $GITHUB_OUTPUT
+      # - name: Configure variables
```

Comment:
are all these lines commented out for testing?

Reply:
yup, I am testing this PR again because I realize that this could be fixed by simply `s/DEVERLOPER_DIR/XCODE_VERSION/`.

If that's the case, we could defer the version upgrade to a different PR down the line.

Reply:
Fixed